index.js

```diff
@@ -51,12 +57,14 @@ const { url, write, file } = parser.parseArgs();
     });
 
   // Handle output
-  if (write) {
-    let content = 'module.exports = [\n';
+  let content = '';
     photos.forEach((photo, index) => {
-      content += '  ' + '\'' + photo + '\'';
+      content += '  ' + '\'' + photo + (maxWidth != null || maxHeight != null ? '=' : '' ) + ( maxWidth != null ? ('w' + maxWidth) : '' ) + (maxWidth != null && maxHeight != null ? '-' : '' ) + ( maxHeight != null ? ('h' + maxHeight) : '' ) + '\'';
```

Comment:
Whoa there :cowboy_hat_face:

Can you break up this line into <80 char lines please?